app/frontend/styles/_header.scss

```diff
@@ -0,0 +1,170 @@
+/* ==========================================================================
```

Comment:
We already include the package in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/package.json#L5. Do we need to import this specifically?

Reply:
Good point - will try to load it from source. In other places they are copied over, but I think it makes sense to load them from node_modules.

Reply:
Done.